src/tests/api/StubTests.test.ts

```diff
@@ -95,5 +96,12 @@ describe("Post Office Stub", () => {
 		const response = await postPOCodeRequest(poStubDelimitator, poStubPayloadData);
 		expect(response.status).toBe(Number(poStubDelimitator));
 	});
+
+	it("returns 200 and the missing name error object when MNE fed as last 3 chars", async () => {
+		const response = await postPOCodeRequest("MNE", poStubPayloadData);
+		console.log("--------------------------------",response.data)
```

Comment:
Please can you address these failures?

Reply:
I removed the console log, tests passing fine locally and on most recent push so not sure what's happened there